adafruit_hid/keyboard.py

```diff
@@ -17,6 +17,10 @@
 
 from . import find_device
 
+try:
+    from typing import Sequence
+except:  # pylint: disable=bare-except
```

Comment:
Is this better than `except ImportError` for memory reasons?  That would get rid of the pylint disable.

Reply:
Right, it makes the mpy smaller because it doesn't have to include the ImportError string.